apps/src/code-studio/components/header/SignInCallout.jsx

```diff
@@ -100,11 +103,10 @@ export default class SignInCallout extends React.Component {
 
   render() {
     return (
-      <div style={styles.container}>
+      <div style={styles.container} onClick={this.props.handleClose}>
         <div
           className="login-callout modal-backdrop"
```

Comment:
I remember why I added this here- we want to specifically test clicks on the modal backdrop, because using the root class that's defined higher up the chain would result in clicks to the sign in button, not the background, and we want to test that clicking the background dismisses it. With that in mind, does it make sense to keep here? If so, I'll definitely update the name.

Reply:
Or, I realize I could nest these 3 siblings under a className, but it doesn't seem like good practice to add another div level just for testing.

Reply:
Ah, I see.  If you specifically want to test clicking the modal backdrop, then I think it'd be ok to just reference the `modal-backdrop` class in the test (and remove `login-callout`).

Reply:
@jamescodeorg Okay- will update this now. I realize though that I should test that clicking the box doesn't redirect to sign in (and instead dismisses, just like the backdrop). If I want to test these both explicitly and independently, then is it worth adding a className to the callout div as well?